data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  values('Agumaon','02-03-2020',0,true,10.23), 
```

Comment:
- [ ] Please remove this unnecessary comma after the insertion because it gives a syntax error when you run the code.